### DISCUSSION TITLE
Will there ever be support for MacOS?

### DISCUSSION THREAD

**Discussion by Furyfree** [ChrisTitusTech/linutil#518] [Q&A]
MacOS is UNIX based and has a package manager. Will there ever be support for that or is it too complicated as there are too many things not needed for MacOS and it's basically only the Application setups that would be relevant?

**Comment by tuxmanxd** [comment]
it is called linutil for a reason

**Comment by cartercanedy** [comment]
This is theoretically possible since everything *should* be implemented as posix compat as possible, but we'd have to carve out a subroutine at startup to check if we're running on a Mac and make sure that xcode tools are installed and `brew` is executable as a prerequisite. It's light work, but I don't think that it's the direction we would want to go
